### PR TITLE
TestFairy documentation.

### PR DESCRIPTION
**Pull request by testfairy**

### CODE REVIEW

**Comment by joshk** [comment]
@plaindocs can you please review this PR when you have a moment

**Comment by plaindocs** [comment]
Roger that.

**Comment by plaindocs** [comment]
@testfairy am I right in thinking that the Symbols, Invites, etc sections apply to both Android and IOS?

**Comment by plaindocs** [comment]
Hi @testfairy ,

I've had a look at this and have a couple of relevant commits on a branch:

* 98cf213447cd5983b9a87aa3e16c139f48305227
* 44baf203d35e509e21210d3c54fd682e8d658107

Could you possibly [sync](https://help.github.com/articles/syncing-a-fork/) this up so I can merge it in?

**Comment by plaindocs** [comment]
@joshk want me to merge this myself, or happy to leave it till @testfairy check back?

**Comment by giltsl** [comment]
@plaindocs sorry for the non comments from @testfairy (@testfairy is an organization and not a GitHub user)
I will create a new PR for the TestFairy documentation (new fork), it will be easier to merge.

**Comment by giltsl** [comment]
Hey @plaindocs I merged `testfairy-gh-pages` into `testfairy/gh-pages`, and resolved the conflicts.
I would be very happy if it will be merged.

Please let me know if something missing.

**Comment by plaindocs** [comment]
Sure thing, will check this and get it in over the next few days.

**Comment by plaindocs** [comment]
Hi @giltsl got a couple more questions inline as I'm unfamiliar with Android dev.

File: user/deployment/testfairy.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+``` yaml
+deploy:
+  provider: testfairy
+  api-key: "TESTFAIRY API KEY"
+  app-file: Path to the app file (APK/IPA)
+  keystore-file: Path to your keystore-file
+  storepass: storepass
+  alias: alias
+```
```
Comment (plaindocs) [inline user/deployment/testfairy.md]: we should call the file the certificate if that is what it is. What is the alias? is it required? The pass is the pass to the ecrypted file?

Comment (giltsl) [inline user/deployment/testfairy.md]: `keystore`, `storepass` and `alias` are the names that's been used in the android documentation.

https://developer.android.com/studio/publish/app-signing.html

**Comment by plaindocs** [comment]
Thanks for your patience on this!

**Pull request merged**